src/charm.py

```diff
@@ -129,21 +140,10 @@ def _on_config_changed(self, event: ConfigChangedEvent) -> None:
         # TODO DPE-5235 support updating data-integrator clients to have/not have public IP
         # depending on the result of the configuration
 
-    def _on_mongos_pebble_ready(self, event) -> None:
-        """Configure MongoDB pebble layer specification."""
-        if not self.is_integrated_to_config_server():
-            logger.info(
-                "mongos service not starting. Cannot start until application is integrated to a config-server."
-            )
-            return
-
-        # Get a reference the container attribute
-        container = self.unit.get_container(Config.CONTAINER_NAME)
+    def _configure_layers(self, container: Container):
```

Comment:
```suggestion
    def __configure_layers(self, container: Container) -> None:
```
only because thats how it is in the VM charm

Reply:
502f904